pywbem_mock/_inmemoryrepository.py

```diff
@@ -0,0 +1,264 @@
+#
+# (C) Copyright 2020 InovaDevelopment.comn
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+"""
+Implementation of the BaseRepository abstract classes that defines an in-memory
+repository based on dictionaries.
+"""
+
+from __future__ import absolute_import, print_function
+
+import six
+
+from pywbem import CIMClass, CIMQualifierDeclaration, CIMInstance
+
+from pywbem._nocasedict import NocaseDict
+from pywbem._utils import _format
+
+from ._baserepository import BaseObjectStore, BaseRepository
+from ._utils import _uprint
+
+__all__ = ['InMemoryRepository', 'InMemoryObjectStore']
+
+
+class InMemoryObjectStore(BaseObjectStore):
+    """
+    Derived from the :class:`BaseObjectStore`, this class implements a
+    dictionary based in-memory repository for CIM objects that manages
+    CIM classes, CIM instances, and CIM qualifier declarations.
+
+    Documentation for the methods and properties inherited from
+    ~pywbem_mock:`BaseObjectStore` is also inherited in the pywbem
+    documentation. Therefore the methods in this class have no
+    documentation string.
```

Comment:
Being part of the docstring, the paragraph "Documentation for ..." appears in the generated docs. I don't think it should be in the generated docs and suggest to move this from the docstring to a block comment.

Reply:
DONE